app/commands/beatmaps.py

```diff
@@ -174,3 +176,52 @@ async def fix_beatmapset(context: Context):
             reference=context.message,
             mention_author=True
         )
+
+
+@app.session.commands.register(['modset'], roles=['BAT', 'Admin'])
+async def change_beatmapset_status(context: Context):
+    """<set_id> <status> - Modify a beatmapset status"""
+
+    if len(context.args) < 2 or not context.args[0].isnumeric():
+        await context.message.channel.send(
+            f'Invalid syntax: `!{context.command} <set_id> <status>`',
+            reference=context.message,
+            mention_author=True
+        )
+        return
+
+    set_id = int(context.args[0])
+
+    if context.args[1].isnumeric():
+        status = int(context.args[1])
+        if status not in range(-2, 5):
+            await context.message.channel.send(
+                f'Invalid status! Valid status: Ranked, Loved, Graveyard, numeric value',
+                reference=context.message,
+                mention_author=True
+            )
+            return
+    else:
+        statuses = {'ranked': 1, 'loved': 2, 'graveyard': -2}
```

Comment:
As I said in the [previous pr](https://github.com/osuTitanic/banchobot/pull/113), there are more statuses than this, which you can see [here](https://github.com/osuTitanic/common/blob/61836e6f67789a020a672bec14c43fd3e9271d99/constants/status.py#L4C6-L4C6).
You could perhaps do:
```python
statuses = {status.name:status.value for status in DatabaseStatus}
```
to select all valid statuses.